tests/hilti/ast/basic-module.hlt

```diff
@@ -7,6 +7,6 @@ module Foo {
 
 type X = bool;
 
-declare string foo(real bar);
+function string foo(real bar) {}
```

Comment:
Could you add valid bodies for this one and the ones below?

```suggestion
function string foo(real bar) { return ""; }
```

We are currently not rejecting these, but I filed #1992 to eventually taking care of these.